adoc/ii-Strategy/SA.adoc

```diff
@@ -7,59 +7,44 @@ The strategy elements are typically used to model the strategic direction and ch
+One key benefit of the cloud is that a business no longer needs to plan hardware sizing for the next five or more years.  Larger, faster, or even smaller infrastructure is only one reboot away.  Still, infrastructure optimization ("rightsizing") remains an important consideration, as businesses that actively manage "rightsizing" their environment can cut costs by 30 to 60 percent.
 
 //image::SA-Strategy.png[title="Solution Architecture - {useCase} Strategy", scaledwidth=80%]
 
 
 === Context
 
-There are many benefits gained when moving your SAP workloads to the cloud:
+There are many benefits gained when moving SAP workloads to the cloud:
 
-Quick deployment:: If you need fast application implementation and deployment, the cloud is the best choice. You can set up a cloud environment within a few hours, whereas, in-house IT infrastructure set-up takes days and sometimes months to order it and set up. With the cloud, IT teams can easily configure and manage the setup remotely.
-+    
-This is the area where SUSE's automation solution for SAP can help.
+Quick deployment:: If you need fast application implementation and deployment, the cloud is the best choice. You can set up a cloud environment within a few hours, whereas, in-house IT infrastructure can take days or even  months to order, install, configure, and bring online.  With the cloud and SUSE's automation solution for SAP, IT teams can easily configure and manage the setup remotely.
 
-Reduce Costs:: Many businesses spent a lot of their capital on maintaining their IT Infrastructure. With help of the cloud, they can transform their CapEx (Capital Expenditure) to stable OpEx (Operating Expenditure).
-Costs can be controlled as the cloud model is a ‘pay as you use’ scenario and many cloud vendors provide several consumption options to provide more choice to their customers. 
-+    
-This is an area where SUSE can help together with the cloud providers to offer the right options, but this also means you need to adapt to how you use the resources and the SAP software.
+Reduce Costs:: Many businesses experience large CapEx (Capital Expenditure) to maintain IT infrastructure, with purchases often in advance of actual need.  SUSE, together with the cloud providers, help businesses transform to a more efficient OpEx (Operating Expenditure) model, with ‘pay as you use’ options that empower businesses with greater agility.
  
-Scalability & Flexibility:: With the cloud, you can scale up your system as and when required, add or remove resources when the business demands it and as mentioned above, "rightsizing" and instance optimization to provide an efficient solution.
+Scalability & Flexibility:: With ready-to-run SUSE images in the cloud providers' marketplaces, businesses can scale up or scale down resources as required.  "Rightsizing" and instance optimization becomes much easier.
```

Comment:
It now say something different - its not because of "ready-to-run" suse images, its because of the flexibility and possibilities what the cloud offers - so "Scalability & Flexibility" is because of the cloud infra and not because the sue images.
And btw. the images are not "ready to run"  - they are as small as possible (just enough os) and must be tailored to the needs/application/workload.

Reply:
Understood.  Would the PAYG SUSE images facilitate this scalability and flexibility?

Reply:
No - payg and byos - have the same packages - its only a different way of "selling" - technical they are 99.99 identical.

payg support the idea of cloud a bit better, as you simply can consume it and dont need to take care to buy and register a subscription from/with suse.

As said, the images have not much to do with scalability and flexibility (of the cloud)

Reply:
Yes, it is a bit of a stretch.  If a business wants to scale dynamically, though, using PAYG subscriptions might be easier than having to purchase enough subscriptions ahead of time.  I really just want to make sure here that we are able to highlight some advantage SUSE provides for scalability & flexibility.